tsconfig.json

```diff
@@ -1,23 +1,35 @@
 {
-  "compilerOptions": {
-    "target":"ES2018",
-    "module": "commonjs",
-    "lib": ["es2018", "dom"],
-    "declaration": true,
-    "strict": true,
-    "noImplicitAny": true,
-    "strictNullChecks": true,
-    "noImplicitThis": true,
-    "alwaysStrict": true,
-    "noUnusedLocals": false,
-    "noUnusedParameters": false,
-    "noImplicitReturns": true,
-    "noFallthroughCasesInSwitch": false,
-    "inlineSourceMap": true,
-    "inlineSources": true,
-    "experimentalDecorators": true,
-    "strictPropertyInitialization":false,
-    "typeRoots": ["./node_modules/@types"]
-  },
-  "exclude": ["cdk.out"]
-}
+    "compilerOptions": {
```

Comment:
if reformatting is checked in, I propose to use the rule of a separate commit for reformatting, otherwise it is hard to see what changed. @kcoleman731 @parkand1